README.md

```diff
@@ -27,3 +27,10 @@ REST API
 ========
 
 [QTI Test REST API](https://openapi.taotesting.com/viewer/?url=https://raw.githubusercontent.com/oat-sa/extension-tao-testqti/master/doc/swagger.json)
+
+E2E testing
+===========
+
+[E2E core](https://github.com/oat-sa/e2e-runner/pull/1) should be installed before running tests
```

Comment:
the views/cypress.env.json should not be committed,
please just add the information inside the readme so everyone can have their own user and pass